tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts

```diff
@@ -94,9 +94,7 @@ export default {
                 Sr25519: "Sr25519Signature",
                 Ecdsa: "EcdsaSignature",
                 Ethereum: "EthereumSignature",
-                EthereumPrettified: "EthereumSignature",
                 Bitcoin: "BitcoinSignature",
-                BitcoinPrettified: "BitcoinSignature",
             },
```

Comment:
opportunistic clean-up: I don't see much value in the Prettified variations, so I removed them. My take is: either we add *Prettified to all of them or to none (Substrate and Evm don't have it, yet they support prettified messages signatures)

Reply:
Yes - or use the way that Kasper suggested - again it's better to do it separately (together with enum change as it will be breaking)

Reply:
I already managed the breaking change on IDH (the only one impacted by this) so we can proceed with it 👍